tenant/src/main/protobuf/com/improving/app/tenant/domain/improving-app-tenant-domain.proto

```diff
@@ -12,6 +12,17 @@ option (scalapb.options) = {
   no_default_values_in_constructor: true
 };
 
+message EstablishTenant {
+  com.improving.app.common.domain.TenantId tenant_id = 1;
+  com.improving.app.common.domain.MemberId creating_user = 2;
```

Comment:
name creating_user as establishing_user for consistency?

Reply:
updated to have establishing_user. Also updated same thing for ActivateTenant and SuspendTenant.